sb_vision/vision.py

```diff
@@ -77,7 +77,7 @@ def process_image(self, img: Image.Image) -> List[Token]:
         img = self.threshold_image(img)
 
         camera_model = self.camera.camera_model
-        detections = list(self.apriltag_detector.detect_tags(img))
+        detections = self.apriltag_detector.detect_tags(img)
```

Comment:
I think Jake's point was that we could leave this in the comprehension below entirely. This would both minimise the diff and also keep the clarity that the previous spelling had.